common/constants/constants.go.testnet

```diff
@@ -45,7 +45,7 @@ var TIPXDCX = big.NewInt(23779191)
 var TIPXDCXLending = big.NewInt(23779191)
 var TIPXDCXCancellationFee = big.NewInt(23779191)
 var TIPXDCXCancellationFeeTestnet = big.NewInt(23779191)
-var TIPXDCXDISABLE = big.NewInt(61290000) // Target 31st March 2024
+var TIPXDCXMinerDisable = big.NewInt(61290000) // Target 31st March 2024
```

Comment:
what about TIPXDCXReceiverDisable in *.testnet and *.devnet?

Reply:
ohoh, forget to add, nice catch.

Reply:
nice catch!